site/content/3.12/release-notes/version-3.12/whats-new-in-3-12.md

```diff
@@ -182,6 +182,31 @@ less overhead.
 
 See [Document and object functions in AQL](../../aql/functions/document-object.md#parse_collection).
 
+### Parallel execution within an AQL query
+
+The new `async-prefetch` optimizer rule allows certain operations of a query to
+asynchronously prefetch the next batch of data while processing the current batch,
+allowing parts of the query to run in parallel. This can lead to performance
+improvements if there is still reserve (scheduler) capacity.
+
+The new `Par` column in a query explain output shows which nodes of a query are
+eligible for asynchronous prefetching. Write queries, graph execution nodes,
+nodes inside subqueries, and query parts involving remote execution are not
+eligible.
```

Comment:
Based on the description of the upstream PR, it sounds like effectively everything REMOTE doesn't use prefetching - but please clarify if this is actually the case

Reply:
All queries parts that include a REMOTE node currently do not use async prefetching. The upmost part of a query (when looking at an execution plan) that does not contain a REMOTE node however can use async prefetching.

In addition, LIMIT nodes currently also do not use async prefetching for themselves or their (upward) dependencies.